tools/crm_resource_runtime.c

```diff
@@ -652,8 +652,20 @@ cli_resource_delete(cib_t *cib_conn, crm_ipc_t * crmd_channel, const char *host_
         }
 
         printf(", removing %s\n", attr_name);
+
+#if HAVE_ATOMIC_ATTRD
         rc = attrd_update_delegate(NULL, 'D', host_uname, attr_name, NULL, XML_CIB_TAG_STATUS, NULL,
                               NULL, NULL, node ? is_remote_node(node) : FALSE);
+#else
+        if (node && is_remote_node(node)) {
+            /* Talk directly to cib for remote nodes if it's legacy attrd */
+            rc = delete_attr_delegate(cib_conn, cib_sync_call, XML_CIB_TAG_STATUS, node->details->id, NULL, NULL,
+                                  NULL, attr_name, NULL, FALSE, NULL);
+        } else {
+            rc = attrd_update_delegate(NULL, 'D', host_uname, attr_name, NULL, XML_CIB_TAG_STATUS, NULL,
+                                  NULL, NULL, node ? is_remote_node(node) : FALSE);
+        }
+#endif
```

Comment:
I think I'd prefer to functionize this so we can avoid duplicating the attrd_update_delegate() code (ie if !atomic ... delete_attr_delegate, return)